CI.hs

```diff
@@ -413,16 +428,16 @@ buildDists ghcFlavor noGhcCheckout noBuilds versionSuffix = do
 
   system_ $ "cd examples/ghc-lib-test-mini-hlint && cabal test --project-dir ../.. --test-show-details direct --test-options \"--color always --test-command ../../ghc-lib-test-mini-hlint " ++ ghcFlavorArg ++ "\""
   system_ $ "cd examples/ghc-lib-test-mini-compile && cabal test --project-dir ../.. --test-show-details direct --test-options \"--color always --test-command ../../ghc-lib-test-mini-compile " ++ ghcFlavorArg ++ "\""
-  system_ "cabal exec -- ghc -ignore-dot-ghci -package=ghc-lib-parser -e \"print 1\""
-  system_ "cabal exec -- ghc -ignore-dot-ghci -package=ghc-lib -e \"print 1\""
+  system_ "cabal -v0 exec -- ghc -ignore-dot-ghci -package=ghc-lib-parser -e \"print 1\""
+  system_ "cabal -v0 exec -- ghc -ignore-dot-ghci -package=ghc-lib -e \"print 1\""
```

Comment:
when testing with ghc-9.12.1, cabal likes to print warning messages about how it hasn't been tested with that version. since we often are trying to parse cabal command output they get in the way so this is a general change that avoids this situation.